Assets/Scripts/Target.cs

```diff
@@ -12,9 +12,14 @@ public class Target : MonoBehaviour
     private float _maxTorque = 10; 
```

Comment:
how would you explain these functions?

try adding a summary to explain the functions, or tooltips.